src/cogent3/util/table.py

```diff
@@ -792,6 +796,8 @@ def _get_repr_(self):
             header = tuple(rows.keys())
         table = self.__class__(header=header, data=rows, **kwargs)
         table._column_templates.update(self._column_templates)
+        if not show_shape:
```

Comment:
I think using show_shape needs to be in the __repr__ and _repr_html_ functions since entire blocks of text will need to be excluded. Just having an empty string I don't think does it -- unless you've check in the interactive interpreter / jupyter?